packages/cli/src/options/beaconNodeOptions/execution.ts

```diff
@@ -80,4 +84,18 @@ export const options: CliCommandOptions<ExecutionEngineArgs> = {
     type: "string",
     group: "execution",
   },
+
+  "jwt-id": {
```

Comment:
I'd suggest we are not adding more flags using this format and start using camelCase for consistency with other CLI flags

```suggestion
  "jwtId": {
```

Maybe it makes sense to also rename the `jwt-secret` flag to `jwtSecret` while keeping previous value as an alias, all docs should reference new flag name

Reply:
yargs automatically populates args with both hyphen case and camel case, kinda an automagic feature iirc. So this can be done and it won't be breaking.

Reply:
That's good to know, if it doesn't do that we can always just add an alias
https://github.com/ChainSafe/lodestar/blob/4a25589900c660bd37b15b3be8bedf0672d69340/packages/cli/src/cmds/validator/options.ts#L173

Reply:
> I'd suggest we are not adding more flags using this format and start using camelCase for consistency with other CLI flags
> 
> Maybe it makes sense to also rename the `jwt-secret` flag to `jwtSecret` while keeping previous value as an alias, all docs should reference new flag name

Indeed yargs automatically handles hyphen and camel case. I will use `jwtId` here.

I will also rename `jwt-secret` to `jwtSecret` in this PR since it's a small change and wouldn't break anything